frontend/src/modules/settings/settings-pricing-plans.js

```diff
@@ -11,46 +12,131 @@ export const plans = {
       description: 'Understand & manage your community',
       price: 'Free',
       features: [
-        '5 seats',
-        'Unlimited community members, organizations & activities',
-        'Community management',
-        'Community intelligence',
-        'Full API access & native integrations with GitHub, Discord, Slack, Twitter, DEV, Reddit, and Hacker News',
-        '5 member enrichments per month (manual)',
-        '2 active workflows & CSV exports per month',
-        'Community & email support',
+        'Unlimited seats',
+        'Unlimited contacts, organizations & activities',
+        '1K monthly active contacts',
+        'Get data from GitHub, Discord, Slack, Discourse, DEV, Reddit, Stack Overflow, Hacker News, Zapier, n8n & more',
+        '2 active automations & CSV exports',
+        'Sentiment analysis',
+        'Full API access',
+        'Community & in-app support',
       ],
+      ctaLabel: {
+        [Plans.values.eagleEye]: 'Downgrade to Essential',
+        [Plans.values.growth]: 'Downgrade to Essential',
+        [Plans.values.scale]: 'Downgrade to Essential',
+        [Plans.values.enterprise]: 'Downgrade to Essential',
```

Comment:
not sure if the `ctaLabel` and `ctaAction` are good, considering they include growth. Also, it's odd that regardless of the user's plan, all the labels read Downgrade to Essential.